builtin/providers/aws/resource_aws_security_group_rule_test.go

```diff
@@ -256,6 +256,92 @@ func TestAccAWSSecurityGroupRule_SelfReference(t *testing.T) {
 	})
 }
 
+// testing partial match implementation
+func TestAccAWSSecurityGroupRule_PartialMatching_Basic(t *testing.T) {
```

Comment:
I think the convention (at least based on quick `grep`) is `_basic` (small caps).